sass/components/_on-this-page.scss

```diff
@@ -0,0 +1,36 @@
+/// internal page navigation
+/// i.e., the "On This Page" section
+
+
+.tree-menu .on-this-page {
```

Comment:
nav.on-this-page was originally to gain more specificity because .media-content was winning but now that i've removed .media-content it's not a problem.